tests/integration_test/data/single_app_as_job/test_ha_2_servers_before_first_round.yml

```diff
@@ -0,0 +1,12 @@
+system_setup: ./data/system/ha_2_servers_2_clients.yml
+apps_root_dir: ./data/apps
+single_app_as_job: True
+cleanup: True
+
+
+tests:
+  - app_name: np_sag
+    validators:
+      - path: tests.integration_test.validators.LogResultValidator
+        args: { expected_in_result: "Round 0 started" }
```

Comment:
I think it is better to also add the following into validators to make sure it does restart and it does finish

      - path: tests.integration_test.validators.NumpySAGResultValidator
        args: { expected_result: [ [ 4, 5, 6 ], [ 7, 8, 9 ], [ 10, 11, 12 ] ] }